In [1]:
from glob import glob
import os
import pandas as pd

In [2]:
filepaths = glob('./Statements/no_stop-phrases/*/*')
years = glob('./Statements/no_stop-phrases/*')

In [3]:
print('Number of total documents: {}'.format(len(filepaths)))

Number of total documents: 218


In [4]:
print("First document: {}\nLast document: {}".format(filepaths[0], filepaths[-1]))

First document: ./Statements/no_stop-phrases\1994\19940204.txt
Last document: ./Statements/no_stop-phrases\2023\20230322.txt


### Search query: 1990-01-01 ~ 2023-05-24
* Period of the scraped documents: 1994-02-04 ~ 2023-03-22
* Total number of statements: 218

In [5]:
print('The number of documents per year is as follows:')
records = []
for year in years:
    records.append((os.path.basename(year), len(glob(os.path.join(year, '*')))))
    
data = pd.DataFrame(records, columns=['year', 'number_of_documents'])
data

The number of documents per year is as follows:


year  number_of_documents
0   1994                    6
1   1995                    3
2   1996                    1
3   1997                    1
4   1998                    3
5   1999                    6
6   2000                    8
7   2001                   11
8   2002                    8
9   2003                    8
10  2004                    8
11  2005                    8
12  2006                    8
13  2007                   10
14  2008                   11
15  2009                    8
16  2010                    9
17  2011                    8
18  2012                    8
19  2013                    8
20  2014                    8
21  2015                    8
22  2016                    8
23  2017                    8
24  2018                    8
25  2019                    9
26  2020                   10
27  2021                    8
28  2022                    8
29  2023                    2